CHANGELOG

```diff
@@ -1,3 +1,9 @@
+Qtile X.X.X, released XXXX-XX-XX:
+    * !!! Dependency update !!!
+        - New dependency needs to be installed for qtile to work: dbus_next.
+          Replaces previous reliance on dbus/Glib and allows qtile to use async
```

Comment:
dbus/glib were previously optional dependencies, but this series adds a requirement that dbus_next be installed, even for people who aren't using dbus at all for any widgets. Is there any chance we can make dbus_next optional?

Reply:
Apologies, I may have misremembered, I thought we said it was ok to add as a dependency given it was so light.

I don't have strong feelings one way or the other on this (although I would like Arch to put `dbus-next` in their main repository) so happy to rework if that's your preferred route.

Reply:
Yep, so feel free to ignore me here :)

Reply:
Although it might be nice to make it optional... I'll leave it up to you.

Reply:
FWIW, we can merge this as-is just to get it in so, too. Looks like it just needs a rebase for the changelog?

Reply:
I agree. Probably politer to make it optional. Will update.